pkg/remoteconfig/state/tuf.go

```diff
@@ -209,3 +209,26 @@ func unsafeUnmarshalRoot(raw []byte) (*data.Root, error) {
 	}
 	return &root, err
 }
+
+func unsafeUnmarshalTargets(raw []byte) (*data.Targets, error) {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 Thought:** ‏We don't really test that file, but maybe we should. In another PR though